llvm/docs/LangRef.rst

```diff
@@ -11976,6 +11976,98 @@ Example:
       <result> = icmp ule i16 -4, 5        ; yields: result=false
       <result> = icmp sge i16  4, 5        ; yields: result=false
 
+.. _i_sthreecmp:
+
+'``sthreecmp``' Instruction
```

Comment:
This isn't an instruction, but an intrinsic (that is, a special type of call). I'd look at the documentation for something like llvm.smax or llvm.sadd.sat and follow what it does (and also place it in the same general area).

Reply:
Sweet, refactored it now.